src/show/pages/public/Standings/index.jsx

```diff
@@ -7,8 +7,8 @@ import {
 } from '../../../components';
 
 const Standings = () => {
-  const [constructor, setConstructor] = useState([]);
-  const [driver, setDriver] = useState([]);
+  const [constructors, setConstructor] = useState([]);
+  const [drivers, setDriver] = useState([]);
```

Comment:
You have `constructors`, but you have `setConstructor` (singular). Same for drivers...

Reply:
@MIKALAKE